.github/workflows/ci-test.yml

```
@@ -19,14 +19,27 @@ jobs:
 
     runs-on: ubuntu-22.04
     services:
+
+      mysql:
+        image: mysql:8.0
+        ports:
+          - '3306:3306'
+        environment:
```

actionlint (1.7.7)

unexpected key "environment" for "services" section. expected one of "credentials", "env", "image", "options", "ports", "volumes"

(syntax-check)